controller/raft/mesh/mesh.go

```diff
@@ -817,6 +817,7 @@ func (self *impl) AcceptUnderlay(underlay channel.Underlay) error {
 		}
 
 		peer.Version = versionInfo
+		peer.SigningCerts = []*x509.Certificate{underlay.Certificates()[0]}
```

Comment:
why not take the full chain? is there a reason to take only leaf?

Reply:
Question for @andrewpmartinez maybe. I'm just copying the behavior of the dialing side to the listening side, and didn't dig too deep into what we use those certs for. If no cert is present, we don't update the db, so that's the specific issue I was fixing.